docs/alert.md

```diff
@@ -13,6 +13,23 @@ This is an API that works both on iOS and Android and can show static
 alerts. To show an alert that prompts the user to enter some information,
 see `AlertIOS`; entering text in an alert is common on iOS only.
 
+## Example
+
+<table>
```

Comment:
I've used a table here instead of markup as the markup table styles the first column as 60% wide, instead of 50/50